plugins/modules/silence_info.py

```diff
@@ -29,6 +29,8 @@
 version_added: "1.0"
 extends_documentation_fragment:
   - sensu.sensu_go.auth
+seealso:
+  - module: silence
 options:
   subscription:
     description:
```

Comment:
Would you please update also these lines below:
* L48: List *all* Sensu silence entries
* L52: Fetch *a* specific silence....